src/main/java/org/jenkinsci/plugins/parallel_test_executor/SplitStep.java

```diff
@@ -82,11 +88,15 @@ public DescriptorImpl() {
 
         private static final long serialVersionUID = 1L;
 
-        @Inject private transient SplitStep step;
-        @StepContextParameter private transient Run<?,?> build;
-        @StepContextParameter private transient TaskListener listener;
+        @Inject
```

Comment:
BTW could also fix the deprecated usages here.

Reply:
I'll have a look at it. There are many annotations and extended classes that require a change.
Therefore I propose to do it in another pull request.

Reply:
Sure, in fact I would prefer that since a PR which fixes an actual deprecation is one I am more likely to want to spend time reviewing!

Reply:
See pull request #41.
I would propose you merge #41 first, then I rebase this one on it (merge conflicts due to the formattings) and then this one gets merged.